pkg/polaris/polarisDBMutator.go

```diff
@@ -72,10 +72,29 @@ func GetPolarisDBComponents(baseUrl string, polaris Polaris) (map[string]runtime
 		content = strings.ReplaceAll(content, "${POSTGRES_TYPE}", "external")
 	}
 
-	content = strings.ReplaceAll(content, "${EVENTSTORE_PV_SIZE}", polaris.PolarisDBSpec.EventstoreDetails.Storage.StorageSize)
-	content = strings.ReplaceAll(content, "${MONGODB_PV_SIZE}", polaris.PolarisDBSpec.MongoDBDetails.Storage.StorageSize)
-	content = strings.ReplaceAll(content, "${UPLOAD_SERVER_PV_SIZE}", polaris.PolarisDBSpec.UploadServerDetails.Storage.StorageSize)
-	content = strings.ReplaceAll(content, "${POSTGRES_PV_SIZE}", polaris.PolarisDBSpec.PostgresDetails.Storage.StorageSize)
+	if len(polaris.PolarisDBSpec.EventstoreDetails.Storage.StorageSize) > 0 {
```

Comment:
Same thing as polaris.PolarisSpec.DownloadServerDetails.Storage.StorageSize. No need for an if statement.